Apps/WebClient/test/unit/Services.Test/UserProfileService_Test.cs

```diff
@@ -268,16 +270,18 @@ public void ShouldGetPreference()
             UserPreference dbUserPreference = new UserPreference
             {
                 HdId = hdid,
-                DismissedMyNotePopover = true,
+                Key = "DismissedMyNotePopover",
```

Comment:
Lets rename it to tutorialPopover to make it general

Reply:
Updated